backends/cuda/ceed-cuda.c

```diff
@@ -128,5 +141,5 @@ static int CeedInit_Cuda(const char *resource, Ceed ceed) {
 
 __attribute__((constructor))
 static void Register(void) {
-  CeedRegister("/gpu/cuda", CeedInit_Cuda, 20);
+  CeedRegister("/gpu/cuda/ref", CeedInit_Cuda, 20);
```

Comment:
See above about setting this to a different value to give priority to one version over another.